design model/components.model.json

```diff
@@ -461,6 +461,35 @@
                     ]
                 }
             ]
+        },
+        {
+            "name": "Grid",
+            "properties": [
+            {
+              "name": "Density",
+              "enumType": {
+                "comfortable": "Comfortable",
+                "cosy": "Cosy",
+                "compact": "Compact"
+              },
+              "value": "Comfortable"
+            },
+            {
+              "name": "Features",
```

Comment:
I imagine this would accumulate all features enabled for the grid, with some additional information on the feature (if such information can be extracted).
Like for example: 
Features: [{
"name": "Filtering",
"mode": "excel"
}, {
"name": "Sorting",
"mode": "single"
}
]
If the feature is enabled just for some of the columns but not for all (for example filtering icon is shown just for 1 of the columns) then that information would be stored in the Columns.  For example:
Column: [{
"name": "Column 1",
"type": "string",
"filterable": true
}]

Reply:
ОК so if under features it is valid for all grid columns in case it is a per-column feature, and if in column X, it is valid only for column X